ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -38,18 +38,29 @@ object LedgerApiTestTool {
   private def exitCode(summaries: Vector[LedgerTestSummary], expectFailure: Boolean): Int =
     if (summaries.exists(_.result.isLeft) == expectFailure) 0 else 1
 
-  private def printAvailableTests(config: Config): Unit = {
+  private def print[A](defaults: Seq[A], optionals: Seq[A])(t: A => String): Unit = {
```

Comment:
The function signature is generic, but the implementation is quite specific. Mentioning the tests in the function name is more honest and memorable. Furthermore, it wouldn't shadow `print` from the `Prelude`.

Reply:
Renamed to `printListOfTests`.